charts/opentelemetry-demo/values.yaml

```diff
@@ -615,10 +626,19 @@ jaeger:
     cassandra: false
   allInOne:
     enabled: true
-    args: ["--memory.max-traces", "10000", "--query.base-path", "/jaeger/ui"]
+    args:
+      - "--memory.max-traces"
+      - "10000"
+      - "--query.base-path"
+      - "/jaeger/ui"
+      - "--prometheus.server-url"
+      - 'http://{{ include "otel-demo.name" . }}-prometheus-server:9090'
```

Comment:
For my own knowledge, why does jaeger need the prometheus server?

Reply:
Jaeger has a way to display span metrics in the tool.  To do so, it needs to know where the metrics are.

Reply:
This is the feature specifically that it enables
![Screenshot 2023-01-31 at 12 09 37 PM](https://user-images.githubusercontent.com/1296118/215833122-8dbda52c-55ca-4a11-ad45-8e3604f5d886.png)